src/main/resources/migrations.xml

```diff
@@ -2719,4 +2719,21 @@
 		<!-- Rolling back is not useful -->
 	</changeSet>
 
+	<changeSet id="create-emailDeactivationWarnings-table" author="jose-west">
+		<comment>Add a table to support a robust email deactivation warning mechanism.</comment>
+		<createTable tableName="emailDeactivationWarnings">
+			<column name="personUuid" type="${uuid_type}" >
+				<constraints nullable="false" />
+			</column>
+			<column name="sentAt" type="datetime" />
+		</createTable>
+
+		<addUniqueConstraint tableName="emailDeactivationWarnings" columnNames="personUuid" constraintName="uniqueEmailDeactivationWarningsPerson" />
```

Comment:
Please follow the naming conventions, so `constraintName="UQ_EmailDeactivationWarningsPerson"`.

Reply:
I will change it. Where can I find the naming convention? I followed the naming type that was used the most in the file.

Reply:
The ones you're referring to were by and large dropped in the grand renaming as part of the `id` to `uuid` switch. So in terms of *active* constraints, most follow the convention: unique constraints start with `UQ_`, foreign key constraints start with `FK_`, primary key constraints start with `PK_`; and for good measure, regular indexes start with `IDX_`, and (on the PostgreSQL PoC) full-text indexes start with `FT_` and trigram indexes with `TR_`.
Unfortunately, a few have slipped through the cracks so naming is not 100% consistent.

Reply:
I will keep it in mind, thanks for the explanation.